agent/exporter/src/main/java/com/microsoft/applicationinsights/agent/Exporter.java

```diff
@@ -509,15 +509,29 @@ private void applyMessagingClientSpan(Attributes attributes, RemoteDependencyTel
 
     private static int getDefaultPortForDbSystem(String dbSystem) {
         switch (dbSystem) {
-            // TODO replace these with constants from OpenTelemetry API after upgrading to 0.10.0
-            // TODO add these default ports to the OpenTelemetry database semantic conventions spec
-            // TODO need to add more default ports once jdbc instrumentation reports net.peer.*
-            case "mongodb":
+            // jdbc default ports are from io.opentelemetry.javaagent.instrumentation.jdbc.JdbcConnectionUrlParser
+            // TODO make these ports constants (at least in JdbcConnectionUrlParser) so they can be used here
+            case SemanticAttributes.DbSystemValues.MONGODB:
                 return 27017;
-            case "cassandra":
+            case SemanticAttributes.DbSystemValues.CASSANDRA:
                 return 9042;
-            case "redis":
+            case SemanticAttributes.DbSystemValues.REDIS:
                 return 6379;
+            case SemanticAttributes.DbSystemValues.MARIADB:
+            case SemanticAttributes.DbSystemValues.MYSQL:
+                return 3306;
```

Comment:
Does otel-java have constants for these ports? It will be nice if they do.  Or also where is the spec for defining these port numbers?

Reply:
see the comment above